log-classifier.cabal

```diff
@@ -47,7 +47,7 @@ library
                      , monad-control
                      , transformers-base
                      , unliftio
-                     -- 
+                     --
```

Comment:
Generally, it's best to leave unchanged lines so they don't confuse people.

Reply:
This is not something I've done by hand. Rather, it was done automatically by a tool while I work on the project, I guess by the editor.